src/main/kotlin/nl/info/zac/app/klant/KlantRestService.kt

```diff
@@ -135,8 +139,11 @@ class KlantRestService @Inject constructor(
 
     @PUT
     @Path("personen")
-    fun listPersonen(restListPersonenParameters: RestListPersonenParameters): RESTResultaat<RestPersoon> =
-        brpClientService.queryPersonen(restListPersonenParameters.toPersonenQuery())
+    fun listPersonen(
+        @HeaderParam("X-Verwerking") xVerwerking: String,
```

Comment:
I'm not sure I'm happy with the name of this parameter. Why would we bother using an `x` prefix? 

The reason for the header to be prefixed with `X_` is that it is a non-standard header introduced by the api specification of the BRP protocollering implementation.

Within our application we want to be explicit about the use of this parameter, and that is to pass the `process` as is.

Reply:
Let's leave this parameter as is, let's follow @xiduzo 's reasoning.